sormas-backend/src/main/java/de/symeda/sormas/backend/caze/CaseFacadeEjb.java

```diff
@@ -1335,7 +1335,12 @@ public CaseReferenceDto getReferenceByUuid(String uuid) {
 
 	@Override
 	public CaseDataDto saveCase(@Valid CaseDataDto dto) throws ValidationRuntimeException {
-		return saveCase(dto, true, true);
+		return saveCase(dto, true, true, false, false);
```

Comment:
`internal` should not be set to false here; see line 1455, the method there sets it to `true` as well. If I understand it correctly, `internal` should always be true unless the case comes from an external source (e.g. sormas2sormas). I suggest to ommit the argument here and let the saveCase method in 1455 handle it.